packages/api/src/external/commonwell/proxy/cw-process-request.ts

```diff
@@ -45,13 +51,14 @@ export async function processRequest(req: Request): Promise<Bundle<Resource>> {
   log(`ORIGINAL URL: ${req.url}`);
 
   const { cxId, patientId } = await getPatientAndCxFromRequest(req);
-
   const { resource, count, params } = fromHttpRequestToFHIR(req);
 
   log(
     `UPDATED resource: ${resource} / cx ${cxId} / patient ${patientId} ` +
       `/ count : ${count}, params: ${params.toString()}`
   );
+
+  // TODO: Remove when done comparing
   const rawResources = await queryFHIRServer({
```

Comment:
Missed removing this and the oldBundle below?

Reply:
Huh.. I must've forgot to push my last changes where I cleaned it all up 😬